research-software-directory/RSD-as-a-service
# Setup end to end tests

# Setup end to end tests

Closes #674 

Changes proposed in this pull request:
*  setup end to end tests for RSD using Playwright framework
*  setup github action triggers for all tests, e2e tests will run on PR and merge to main, it uses e2e/.env.e2e file with "fake" secrets
*  renamed `make down` to `make stop`    
*  renamed testing github actions
*  removed unit test from frontend build process
*  created `make e2e-tests` to run tests locally

How to test:
* `make e2e-tests` to run e2e tests locally
*  e2e tests should also run on each PR, so this PR should also run e2e tests :-)

PR Checklist:
